.github/labeler.yml

```diff
@@ -0,0 +1,25 @@
+# https://github.com/actions/labeler#common-examples
+# Adapted from https://github.com/rapidsai/raft/blob/main/.github/CODEOWNERS
+# Labels culled from https://github.com/rapidsai/raft/labels
+
+Python:
+ - 'python/**'
+ - 'notebooks/**'
+
+libraft:
+ - 'cpp/**'
+
+CMake:
+ - '**/CMakeLists.txt'
+ - '**/cmake/**'
+
+Java:
+ - 'java/**'
```

Comment:
These labels should match what labels are in this repo and the setup of the `.github/CODEOWNERS` for this repo